### PR TITLE
Add support multiple entities (actors) of Spriter file.

### PR DESCRIPTION
**Pull request by twachikku**
Add Entity selection box that user can set the entity name of Spriter animation object.

image

### CODE REVIEW

File: plugins/vis-editor-spriter/src/main/java/com/kotcrab/vis/plugin/spriter/ui/VisSpriterComponentTable.java [hunk 1]
```diff
@@ -42,11 +43,17 @@ public VisSpriterComponentTable (ModuleInjector injector) {
...
+		entitySelectBox.getSelection().setProgrammaticChangeEvents(false);
+		entitySelectBox.addListener(properties.getSharedSelectBoxChangeListener());
+		
 		animSelectBox = new VisSelectBox<>();
 		animSelectBox.getSelection().setProgrammaticChangeEvents(false);
 		animSelectBox.addListener(properties.getSharedSelectBoxChangeListener());
 
 		VisTable table = new VisTable(true);
+		table.add(new VisLabel("Entity"));
+		table.add(entitySelectBox);
```
Comment (kotcrab) [inline plugins/vis-editor-spriter/src/main/java/com/kotcrab/vis/plugin/spriter/ui/VisSpriterComponentTable.java]: Add table.row() after `table.add(entitySelectBox);` so UI can fit in Entity Properties panel. 

Also please run our code formatter on code changes (see CONTRIBUTING files).

Comment (twachikku) [inline plugins/vis-editor-spriter/src/main/java/com/kotcrab/vis/plugin/spriter/ui/VisSpriterComponentTable.java]: updated.

**Comment by twachikku** [comment]
In additional;
I modified VisSpriterAnimation component to support a simple grid SpriteSheet (fixed row and column).
image

**Comment by kotcrab** [comment]
Please keep one feature per PR. Even if I could merge Spriter feature it can be blocked by some others. There are some problems with this implementation.
- `SceneLoader`: it is completely OK to add atlas dependency few times, it is internally managed by AssetManager to load it only once.
- You modified Spriter runtime.  (`com.brashmonkey.spriter` package) this will lead to problems when updating. Please check if there is some method to avoid this.
- `row` and `column` should be called `rows` and `columns` in my opinion and have proper tooltip like "Number of sprite sheet columns", columns min should be just `min = 1`
- VisSprite shouldn't use base region, I see what you did it but notice that there would be many more sprites not ever using it than actually it will be used for animations. It would be more fitting to store it in `VisSpriteAnimation` or even separate component.
- Lastly, VisSpriteAnimation component UI can't be left like this. Rows/columns shouldn't be showed when region comes from `TextureAtlas` same for "problems detected".

And please do yourself a favour and setup proper git environment so you don't have to upload every change by web upload, on Windows you can use [Github Desktop](https://desktop.github.com/).

**Comment by twachikku** [comment]
I updated all, except the VisSpriteAnimation component UI.
I don't know how to hide Rows/columns properties when region comes from TextureAtlas.

**Comment by kotcrab** [comment]
You changed line endings which causes diffs in files that wasn't changed (probably missing autocrlf: `git config --global core.autocrlf true`) making it harder to review and later to use tools like `git blame`. You want to fix this or should I just go and apply those changes manually since these are simple features?

**Comment by twachikku** [comment]
Please, fix it for me. Now I'm very confuse, what is my mistake.  I just use the normally Eclipse, and I never changed any setting, but why the line endings was changed.

**Pull request closed without merging**